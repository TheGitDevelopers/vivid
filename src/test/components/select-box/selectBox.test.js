import React from 'react';
import renderer from 'react-test-renderer';
import SelectBox from '../../../components/common/select-box/selectBox';


describe('SelectBox tests:', () => {
  /* Test mockups */
  const mockedList = [];

  const onSelectItemMock = jest.fn();

  const tree = renderer
    .create(<SelectBox onSelectItem={onSelectItemMock} list={mockedList} />);

  /* All component tests should be listed here */
  it('renders correctly', () => {
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
