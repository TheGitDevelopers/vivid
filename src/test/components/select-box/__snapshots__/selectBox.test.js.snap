// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectBox tests: renders correctly 1`] = `
<View
  style={
    Object {
      "width": "100%",
    }
  }
>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#F0F4F8",
        "borderRadius": 6,
        "flexDirection": "row",
        "justifyContent": "center",
        "paddingLeft": 20,
        "paddingRight": 30,
        "width": "100%",
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      editable={false}
      placeholder=""
      rejectResponderTermination={true}
      style={
        Object {
          "color": "#A6BCD0",
          "fontFamily": "lato-regular",
          "fontSize": 16,
          "padding": 14,
          "width": "85%",
        }
      }
      underlineColorAndroid="transparent"
      value=""
    />
    <View
      style={
        Object {
          "transform": Array [
            Object {
              "rotate": "0deg",
            },
          ],
        }
      }
    >
      <Text />
    </View>
  </View>
</View>
`;
